inyoka/wiki/macros.py

```diff
@@ -236,17 +236,22 @@ def build_node(self, context, format):
             # TODO: Can we do something else instead of skipping?
             if ':' not in part:
                 continue
-            key = part.split(':')[0].strip()
-            values = [x.strip() for x in part.split(':')[1].split(',')]
-            mapping.extend([(key, x) for x in values])
+            key, values = part.split(":")
+            values = values.split(",")
+            mapping.extend([(key.strip(), x.strip()) for x in values])
         mapping = MultiMap(mapping)
 
         pages = set()
 
         for key in list(mapping.keys()):
             values = list(flatten_iterator(mapping[key]))
-            includes = [x for x in values if not x.startswith('NOT ')]
-            kwargs = {'key': key, 'value__in': includes}
+            includes = [x for x in values if not x.startswith(('NOT ', 'EXACT',))]
+            kwargs = {'key': key}
+            if values[0].startswith("EXACT "):
+                exact_value = values[0][6:]
+                kwargs['value__iexact'] = exact_value
```

Comment:
`iexact` queries are in general slow. Is case-insensitive matching really needed here?


If yes, we could add an index like for the usernames https://github.com/inyokaproject/inyoka/commit/f62bcb37184d705172d1fb7ff44fc1199559e83d